src/servers/src/metrics_handler.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_telemetry::metric;
+
+/// a server that serves metrics
+/// only start when datanode starts in distributed mode
+#[derive(Clone)]
```

Comment:
ZST could

```suggestion
#[derive(Copy)]
```

to get rid of explicitly calling `clone()` elsewhere.